app/models/submission_cleaner.rb

```diff
@@ -78,10 +78,13 @@ def send_destruction_mail_to_submitters
     I18n.available_locales.each do |l|
       local_submitter_ids = @submitters.where(locale: l).pluck(:id)
       next if local_submitter_ids.empty?
-      NotificationMailer.with(recipients: local_submitter_ids,
-                              term: @previous_term,
-                              locale: l)
-                        .submission_destruction_email.deliver_now
+
+      local_submitter_ids.in_groups_of(200) do |group|
```

Comment:
The last group will contain nil values: 
```ruby
[1,2,3].in_groups_of(20)
 => [[1, 2, 3, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil]]
```
Changing it to `in_groups_of(200,false)` should fix it:
```ruby
[1,2,3].in_groups_of(20,false)
 => [[1, 2, 3]]
```